common/script/statHelpers.js

```diff
@@ -0,0 +1,44 @@
+/*
+  ------------------------------------------------------
+  Level cap
+  ------------------------------------------------------
+ */
+
+export const MAX_LEVEL = 100;
+
+export function capByLevel (lvl) {
+  if (lvl > MAX_LEVEL) {
+    return MAX_LEVEL;
+  } else {
+    return lvl;
+  }
+}
+
+/*
+  ------------------------------------------------------
+  Health cap
+  ------------------------------------------------------
+ */
+
+export const MAX_HEALTH = 50;
+
+/*
+  ------------------------------------------------------
+  Scoring
+  ------------------------------------------------------
+ */
+
+export function toNextLevel (lvl) {
+  return Math.round((Math.pow(lvl, 2) * 0.25 + 10 * lvl + 139.75) / 10) * 10;
```

Comment:
At the very least, we should split this out into multiple declarations. Though, again we have the problem with creating variable names for processes we don't fully understand :/

@Alys Do you know what these numbers signify?